### DISCUSSION TITLE
Spacing between objects and relative alignment

### DISCUSSION THREAD

**Discussion by minutiaes** [uxbox/uxbox#1019] [Q&A] [answered]
Hello, is it possible to set a exact spacing between two objects? When there is a pattern, the value appears like in the screenshot below. However the value stays there when it is in a range of +-1.

![Example](https://user-images.githubusercontent.com/23055660/121019409-b1b3a500-c79f-11eb-8901-d3e294f60c7c.jpg)

Should one calculate the position of objects all the time? Is there a way to align an object relative to another one instead of artboard?

**Comment by myfunnyandy** [comment, marked as answer]
Hi, you can position objects relative to other objects using the aling and distribute buttons, but for the moment you can not set exact spacings.

![image](https://help.penpot.app/img/layers-align.gif)

![image](https://help.penpot.app/img/layers-distribute.gif)
